src/core/api/azureblob/azureblob_authorizative_policy.xml

```diff
@@ -26,6 +26,33 @@
         </when>
       </choose>
 
+    <set-variable name="allowedSize" value="${pgp-put-limit-bytes}" />
+    <set-variable name="bodySize" value="@(context.Request.Headers["Content-Length"][0])" />
+
+    <!-- Check for Content-Length not zero-->
+    <choose>
```

Comment:
@TommasoLencioni Please let's check content length header we are actually receiving from senders

Reply:
@GiovanniMancini checked, we receive Content-Length header from each sender currently onboard 👍